tools/eval_net_motion.py

```diff
@@ -0,0 +1,549 @@
+"""Run inference on test set scans.
```

Comment:
thinking more about it, it may make sense to build your motion pipeline into the original `eval_net.py` file. it looks like the changes should be relatively simple to that file, because most of the heavy lifting is being done by the transform